openedx/core/djangoapps/call_stack_manager/tests/tests.py

```diff
@@ -0,0 +1,188 @@
+"""
+Test cases for Call Stack Manager
+"""
+from mock import patch
+from .models import ModelMixin, ModelNothing, ModelMixinCSM, ModelAnotherCSM, ModelWithCSM, ModelWithCSMChild
+from django.test import TestCase
+from openedx.core.djangoapps.call_stack_manager import donottrack
+
+
+class TestingCallStackManager(TestCase):
+    """
+    Tests for call_stack_manager package
+    """
+    @patch('openedx.core.djangoapps.call_stack_manager.core.log.info')
+    def test_save(self, log_capt):
+        """ tests save functionality of call stack manager/ applies same for delete()
+        1. classes with CallStackMixin should participate in logging.
+        """
+        ModelMixin(id_field=1).save()
+        desired_class = ModelMixin
+        self.assertEqual(desired_class, log_capt.call_args[0][1])
```

Comment:
Nit: I'd probably just put `ModelMixin` directly in the `assertEqual` call, rather than using the `desired_class` variable.  It cuts down on the line-noise.